src/main/java/nwoolcan/view/warehouse/article/NewArticleModalController.java

```diff
@@ -37,19 +37,15 @@
 
     /**
      * Creates itself and inject the controller and the view manager.
-     *
      * @param controller  injected controller.
      * @param viewManager injected view manager.
      */
     public NewArticleModalController(final Controller controller, final ViewManager viewManager) {
         super(controller, viewManager);
     }
 
-    /**
-     * Initialize the view create article.
-     */
     @FXML
-    public void initialize() {
+    private void initialize() {
         newArticleUnitOfMeasure.getItems().setAll(UnitOfMeasure.values());
```

Comment:
this is not how quantities and units of measure work.
not all units of measure can be a quantity.
to know who, you must call `Quantities.getValidUnitsOfMeasure()` (don't remember the correct name of the method)

Reply:
That is already in master, open an issue and i will fix it